communication/mailing-list-guidelines.md

```diff
@@ -1,293 +1,270 @@
 
-Moderators should adhere to the general Kubernetes project
-[moderation guidelines].
-
+Moderators should adhere to the general Kubernetes project 
+[moderation guidelines][].
 
 #### New user posting queue
 
 New members who post to the mailing list will automatically have their messages
-put in the [moderation queue]. Moderators of the list will receive a
+put in the moderation queue. Moderators of the list will receive a
 notification of their message and should process them accordingly.
 
-
 #### Annual permissions review
 
-SIG, WG, and subproject Moderators must establish an annual review of their 
+SIG, WG, and subproject Moderators must establish an annual review of their
 mailing lists to ensure their Moderator list is current and includes 
-[mailing list owners]. Many of the SIG and WG mailing lists pre-date current 
+[mailing list owners][]. Many of the SIG and WG mailing lists pre-date current
 communication policy and an annual review ensures ownership is up to date.
 
 This review does not need to occur at a specific recurring date and can be
-combined with other actions such as SIG/WG/subproject leadership changes or 
-sub-project additions.
-
-
----
+combined with other actions such as SIG, WG, and subproject leadership changes
+or subproject additions.
 
 ## Mailing list creation
 
-All SIGs and WGs require two discussion groups: one for leads/chairs, and one 
-for members. Subprojects that opt to have a mailing list only require one for 
-members.
+All SIGs and WGs require two discussion groups: one for leads and chairs, and
+one for members. Subprojects that opt to have a mailing list only require one
+for members.
 
 ### Prerequisites for creating a mailing list
 
-- An email account that can create Google Groups and add members external to 
-your organization to a Google Group mailing list. **This might not be possible 
-with your employer's email account**. You might need to use a personal email 
-account.
-- At least 3 mailing list owners (leads), in addition to 
-contributors@kubernetes.io
-- Familiarity with the [moderation guidelines] for the project and 
-[moderation queue]s. Chairs should be cognizant that a new group will require
-an initial time investment moderation-wise as the group establishes itself.
+-   An email account that can create Google Groups and add members external to
+    your organization to a Google Group mailing list. **This might not be
+    possible with your employer's email account**. You may need to use a
+    personal email account to have access to this function.
 
+-   At least 3 mailing list owners (leads), in addition to
+    contributors@kubernetes.io
+
+-   Familiarity with the [moderation guidelines][] for the project and
+    [moderation queues][moderation queue]. Chairs should be cognizant that a
+    new group will require an initial time investment moderation-wise as the
+    group establishes itself.
 
 ### Create the leads and members mailing lists
```

Comment:
@mrbobbytables should we be instructing people to create lists under kubernetes.io now?

Note:  the instructions below this are just reformatting of the instructions already there, so it shouldn't hold up the PR approval.  However, it raises a question for future revisions.

Reply:
I'd do that in a separate commit or separate PR from the style edits, I wouldn't necessarily block on it.

Reply:
Right.  More of a flag as a TODO.  Lemme open an issue.